swap/Cargo.toml

```diff
@@ -77,9 +79,12 @@ zip = "0.5"
 bitcoin-harness = "0.2.2"
 get-port = "3"
 hyper = "0.14"
+jsonrpsee = { version = "0.16.2", features = [ "ws-client" ] }
+jsonrpsee-types = { version = "0.16.2" }
```

Comment:
I removed jsonrpsee-types since we don't use it anymore, but here we are pulling the WebSocket client feature of jsonrpsee which we use for tests only